tree/ntuple/v7/inc/ROOT/RNTupleIndex.hxx

```diff
@@ -126,6 +129,10 @@ private:
    /// Only built indexes can be queried.
    bool fIsBuilt = false;
 
+#ifdef R__USE_IMT
+   std::unique_ptr<ROOT::TThreadExecutor> fPool;
+#endif
```

Comment:
Is this member required? AFAICT it's only used once during `Build()`, so it might be constructed and destructed locally